protocol/yggdrasil.proto

```diff
@@ -67,3 +71,9 @@ message Data {
 
 // A Receipt message is sent as a successful response to a Send method.
 message Receipt {}
+
+// A DisconnectionResponse message contains the result of a disconnection request.
+message DisconnectionResponse {
+    // Whether or not the worker accepted the disconnection request.
+    bool disconnected = 1;
```

Comment:
In what scenario the worker may reject disconnection request? 
Both yggdrasil and workers mustn't refuse to disconnect from the control plane.

Reply:
So what should be the response?

Reply:
Maybe empty, like `Receipt`?